README.md

```diff
@@ -61,7 +57,6 @@ My first project at Microverse
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
 
 - **[key_feature_1]**
 - **[key_feature_2]**
```

Comment:
- [ ] In the Key Features section, kindly add 1-3 things that you accomplished in this project. I would also advise you to get used to adding these features because you would do that for every project in Microverse and possibly in the professional world. 👍